kubemarine/resources/schemas/definitions/services/kubeadm_patches.json

```diff
@@ -0,0 +1,250 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "description": "Override the original settings for patches of control-plane pods and kubelet",
+  "properties": {
+    "apiServer": {
+      "$ref": "#/definitions/ApiServer"
+    },
+    "etcd": {
+      "$ref": "#/definitions/ETCD"
+    },
+    "controllerManager": {
+      "$ref": "#/definitions/ControllerManager"
+    },
+    "scheduler": {
+      "$ref": "#/definitions/Scheduler"
+    },
+    "kubelet": {
+      "$ref": "#/definitions/Kubelet"
+    }
+  },
+  "definitions": {
+    "ApiServer": {
+      "type": "array",
+      "description": "Patches for kube-apiserver",
+      "minItems": 1,
+      "items": [
+      {
+        "type": "object",
+        "properties": {
+          "groups": {
+            "$ref": "../common/node_ref.json#/definitions/Roles",
+            "description": "The list of group names to whose hosts the patch should be uploaded"
+          },
+          "patch": {
+            "type": "object",
+            "properties": {
+              "additionalProperties": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      {
+        "type": "object",
+        "properties": {
+          "nodes": {
```

Comment:
Probably makes sense to verify nodes have correct roles (in code)

Reply:
done